app/vmalert/datasource/vm_prom_api.go

```diff
@@ -22,6 +20,9 @@ type promResponse struct {
 		ResultType string          `json:"resultType"`
 		Result     json.RawMessage `json:"result"`
 	} `json:"data"`
+	Stats struct {
+		SeriesFetched int64 `json:"seriesFetched,omitempty"`
+	} `json:"stats,omitempty"`
```

Comment:
i think it is better to create separate struct for better code readability

Reply:
Got it, but I see other fileds like `promResponse.Data` and `promInstant.Result` around `Stats` also not separate struct, should I separate them all?

Reply:
all fixed